package.json

```diff
@@ -6,12 +6,13 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "d3": "^6.2.0",
+    "mapbox-gl": "^1.3.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.3",
     "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^2.0.0",
-    "mapbox-gl": "^1.3.1"
+    "semantic-ui-react": "^2.0.0"
```

Comment:
okay there's some change, not sure whether it will create any conflict when we merge, but I think we should be fine since contents are the same